lib/plottr_import_export/src/importer/scrivener/importer.js

```diff
@@ -645,17 +757,108 @@ function mapMatchedFiles(newJson, matchFiles, fileContents, bookId, isSection) {
         // notes.rtf has content other than plotline
         // sections (chars/places/notes) are restricted to use the notes.rtf values
         if (newRTFContent.contents && !isSection) {
-          fileContents.rtfContents.contents = newRTFContent.contents
+          fileContents.rtfContents = newRTFContent
         }
       } else {
-        const newRTFContent = getRTFContents(newJson.lines, file, matchedFileId, bookId, isSection)
+        const newRTFContent = getRTFContents(
+          newJson,
+          newJson.lines,
+          file,
+          matchedFileId,
+          bookId,
+          isSection
+        )
         Object.assign(fileContents.rtfContents, newRTFContent)
       }
     }
   })
   return fileContents
 }
 
+function getManuscriptSectionItems(currentJson, RTFContents, bookId) {
```

Comment:
This function needs tests.
It's riddled with assumptions and heuristics, and we must verify that it works as we intended it to.